pkgs/development/libraries/mongocxx/default.nix

```diff
@@ -49,7 +50,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "Official C++ client library for MongoDB";
     homepage = "http://mongocxx.org";
     license = licenses.asl20;
-    maintainers = with maintainers; [ adriandole ];
+    maintainers = with maintainers; [ adriandole vcele ];
     pkgConfigModules = [ "libmongocxx" "libbsoncxx" ];
     platforms = platforms.all;
     badPlatforms = [ "x86_64-darwin" ]; # needs sdk >= 10.14
```

Comment:
You can override in top-level: `stdenv = darwin.apple_sdk_11_0.stdenv`

Reply:
Ok 👍